content/How to work with Stripe.md

```diff
@@ -0,0 +1,133 @@
+
+Another useful case that is worth mentioning is to apply a coupon to a user, this means that they will have a positive balance for any future invoice (be careful if you charge users with multiple products)
+
+### SUBSCRIPTION ITEMS
+These are your billing items, so for the case of a web subscription you will just have 1 subscription item. For specific cases like an amazon cart or any complicated use case (where you have multiple items being added to a purchase) is where you have to start considering adding some specific logic to your app.
+I won't get really into detail about this, I just wanted to show the general concept behind this, maybe I will write more in detail in a future post.
+
+##### RENEWALS
+Don't overthink it, there is a webhook for most of your use cases. But for this specific need you can configure the following events:
+
+ - customer.subscription.updated
+This event happens every time a susbscription is updated according to this [documentation](https://stripe.com/docs/billing/subscriptions/change)
+
+ - customer.subscription.deleted
+As simple as it sounds, it tells you when a subscription is canceled so you can take the actions needed in your app (possibly dissable the associated account)
+
+ - invoice.payment_succeeded
+This is a really important one! It tells us when a payment is actually accepted by the credit card provider (some times they can be fraud or declined)
+
+### WEBHOOKS
+There are a lot of them and they will solve most of your problems, the only downcase is the headache trying to understand which exactly to use.
+I'm sorry to dissapoint you if you reached here trying to answer this question but up to now I only know [this page](https://stripe.com/docs/api/events/types) that explains the different existing webhooks and what they do. The other option is when you go to create a webhook from the developer's Stripe dashboard, they explain a bit more in detail what each event does.
+
+## 4) SPECIAL RECOMMENDATIONS FOR FURTHER PAYMENT IMPLEMENTATION
+Keep these Stripe documentation pages as your friends:
+ - [Devs api](https://stripe.com/docs/api/)
+ - [Events types](https://stripe.com/docs/api/events/types)
+
+Sometimes there are two or even three ways of solving a problem, so consider this and take your time to analyze each requirement properly before you start coding.
+
+## 5) CONCLUSIONS
+You can easily add online payments to your app and test it in just 1 week (or so), thats amazing! The other amazing thing is that you can start managing most of the daily based situations like fraud of disputes just from the dashboard (you don't need to keep coding).
+
+The difficult part of this is when you start adding more concrete and detailed transactions and supporting multiple transfers types (like bank account tranfers instead of just Visa or MasterCard). So if you liked this post and want to know more don't hesitate to leave some comments asking for it!
```

Comment:
```suggestion
The difficult part of this is when you start adding more concrete and detailed transactions and supporting multiple transfer types (like bank account transfers instead of just Visa or MasterCard). So if you liked this post and want to know more don't hesitate to leave some comments asking for it!
```